Source/WebKit/WebProcess/Extensions/Interfaces/WebExtensionAPIStorageArea.idl

```diff
@@ -28,17 +28,16 @@
     ReturnsPromiseWhenCallbackIsOmitted,
```

Comment:
Since all the calls need the page now, you can add `NeedsPageWithCallbackHandler` here.